### PR TITLE
use new sonata admin config nodes instead of overwriting template

### PR DESCRIPTION
**Pull request by OskarStark**
simplified docs

### CODE REVIEW

File: docs/reference/formatter_widget.rst [hunk 1]
```diff
@@ -127,81 +126,27 @@ If you stop here, the most interesting part will not be present. Let's edit some
     If you want to see an actual implementation of this can, you can have a look at the ``SonataNewsBundle``.
 
 
-Dynamic Input
```
Comment (jordisala1991) [inline docs/reference/formatter_widget.rst]: Why are you removing this block?

File: docs/reference/formatter_widget.rst [hunk 2]
```diff
@@ -127,81 +126,27 @@ If you stop here, the most interesting part will not be present. Let's edit some
...
-        form:
-            resources:
-                # ...
-                - 'SonataFormatterBundle:Form:formatter.html.twig'
-
-
-Make sure the ``SonataFormatterBundle:Form:formatter.html.twig`` is set.
-This template contains custom javascript code to load the selected text editor.
-
-You also need to include some assets in your template:
```
Comment (jordisala1991) [inline docs/reference/formatter_widget.rst]: Probably this is what you want to remove

File: docs/reference/formatter_widget.rst (hunk 1 above)
Comment (OskarStark) [reply docs/reference/formatter_widget.rst]: IMO it is duplicate and the same to achieve both

Comment (OskarStark) [reply docs/reference/formatter_widget.rst]: all the stuff is already named in the installation section, so it is duplicate content
https://github.com/sonata-project/SonataFormatterBundle/blob/3.x/docs/reference/installation.rst#configuration

**Comment by jordisala1991** [comment]
Thank you @OskarStark

**Pull request merged**